lib/modules/nightmode.css

```diff
@@ -201,15 +201,19 @@
 .res-nightmode .arrow.upmod,
 .res-nightmode .arrow.down,
 .res-nightmode .arrow.downmod {
-	background-image: url(https://b.thumbs.redditmedia.com/WZDoQsDzUySVTErcweuA3k2HH1HU5-BZNnBXOR4FDwc.png);
+	width: 15px;
+	height: 14px;
+	margin-left: auto;
+	margin-right: auto;
+	background-image: url(http://i.imgur.com/o1IiV1o.png) !important;
```

Comment:
Let's host the spritesheet on r/RESUpdates/about/stylesheet to keep our dependencies consistent.